### PR TITLE
Stop rewriting -> & => with scalariform

### PR DESCRIPTION
**Pull request by pdalpra**

### CODE REVIEW

File: src/main/scala/tv/teads/github/api/model/DateTimeCodecs.scala [hunk 1]
```diff
@@ -10,11 +10,11 @@ object DateTimeCodecs {
...
-    _.as[Int].map(dayNumber ⇒ DayOfWeek.of(dayNumber + 1).minus(1))
+    _.as[Int].map(dayNumber => DayOfWeek.of(dayNumber + 1).minus(1))
   )
 
   implicit final val TimestampSecondsDecoder = Decoder.instance[ZonedDateTime](
-    _.as[Long].map(Instant.ofEpochSecond).map(instant ⇒ ZonedDateTime.ofInstant(instant, UtcZoneId))
+    _.as[Long].map(Instant.ofEpochSecond).map(instant => ZonedDateTime.ofInstant(instant, UtcZoneId))
   )
 
   implicit final val IsoOffsetToZoneDateTimeDecoder = decodeZonedDateTime(DateTimeFormatter.ISO_OFFSET_DATE_TIME)
```
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/model/DateTimeCodecs.scala]: Codacy Issue found: [Field name 'IsoOffsetToZoneDateTimeDecoder' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/dalprapierre_2790/github-api-client_new/file/2529136264/issues/source?bid=3282900&fileBranchId=3282915#l20)

File: src/main/scala/tv/teads/github/api/services/AbstractGithubService.scala [hunk 2]
```diff
@@ -13,7 +13,7 @@ private[services] abstract class AbstractGithubService(config: GithubApiClientCo
 
   protected type EC = ExecutionContext
   protected type FutureResponse = Future[ResponseWrapper]
-  protected type RequestBuilderF = Request.Builder ⇒ Request.Builder
+  protected type RequestBuilderF = Request.Builder => Request.Builder
 
   private val JsonPrinter = Printer.noSpaces.copy(dropNullKeys = true)
```
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/services/AbstractGithubService.scala]: Codacy Issue found: [Field name 'JsonPrinter' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/dalprapierre_2790/github-api-client_new/file/2529136292/issues/source?bid=3282900&fileBranchId=3282915#l18)

File: src/main/scala/tv/teads/github/api/model/DateTimeCodecs.scala (hunk 1 above)
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/model/DateTimeCodecs.scala]: Codacy Issue found: [Field name 'IsoOffsetToZoneDateTimeDecoder' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/dalprapierre_2790/github-api-client_new/file/2529136264/issues/source?bid=3282900&fileBranchId=3282915#l20)

File: src/main/scala/tv/teads/github/api/services/AbstractGithubService.scala (hunk 2 above)
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/services/AbstractGithubService.scala]: Codacy Issue found: [Field name 'JsonPrinter' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/dalprapierre_2790/github-api-client_new/file/2529136292/issues/source?bid=3282900&fileBranchId=3282915#l18)

File: src/main/scala/tv/teads/github/api/model/DateTimeCodecs.scala (hunk 1 above)
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/model/DateTimeCodecs.scala]: Codacy Issue found: [Field name 'IsoOffsetToZoneDateTimeDecoder' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/dalprapierre_2790/github-api-client_new/file/2529136264/issues/source?bid=3282900&fileBranchId=3282915#l20)

File: src/main/scala/tv/teads/github/api/services/AbstractGithubService.scala (hunk 2 above)
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/services/AbstractGithubService.scala]: Codacy Issue found: [Field name 'JsonPrinter' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/dalprapierre_2790/github-api-client_new/file/2529136292/issues/source?bid=3282900&fileBranchId=3282915#l18)

File: src/main/scala/tv/teads/github/api/model/DateTimeCodecs.scala (hunk 1 above)
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/model/DateTimeCodecs.scala]: Codacy Issue found: [Field name 'IsoOffsetToZoneDateTimeDecoder' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/dalprapierre_2790/github-api-client_new/file/2529136264/issues/source?bid=3282900&fileBranchId=3282915#l20)

File: src/main/scala/tv/teads/github/api/services/AbstractGithubService.scala (hunk 2 above)
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/services/AbstractGithubService.scala]: Codacy Issue found: [Field name 'JsonPrinter' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/dalprapierre_2790/github-api-client_new/file/2529136292/issues/source?bid=3282900&fileBranchId=3282915#l18)

**Comment by gbougeard** [comment]
LGTM

File: src/main/scala/tv/teads/github/api/model/DateTimeCodecs.scala (hunk 1 above)
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/model/DateTimeCodecs.scala]: Codacy Issue found: [Field name 'IsoOffsetToZoneDateTimeDecoder' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/dalprapierre_2790/github-api-client_new/file/2529136264/issues/source?bid=3282900&fileBranchId=3282915#l20)

File: src/main/scala/tv/teads/github/api/services/AbstractGithubService.scala (hunk 2 above)
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/services/AbstractGithubService.scala]: Codacy Issue found: [Field name 'JsonPrinter' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/dalprapierre_2790/github-api-client_new/file/2529136292/issues/source?bid=3282900&fileBranchId=3282915#l18)

**Pull request merged**